stylix/testbed.nix

```
@@ -37,6 +37,30 @@ let
       };
     };
 
+  enableModule =
+    { lib, ... }:
+    {
+      options.stylix.testbed.enable = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        example = lib.literalExpression "lib.meta.availableOn pkgs.stdenv.hostPlatform pkgs.discord";
+        description = ''
+          Whether to enable this testbed.
+
+          The testbed will not be included as a flake output if set to false;
+
+          > [!CAUTION]
+          >
+          > This option's value can only evaluate `lib` and `pkgs` inputs,
+          > if it reads other inputs, the testbed will fail to evaluate;
+          > e.g., `config` or `options`.
```

```suggestion
          > [!CAUTION]
          >
          > This option's value can only evaluate `lib` and `pkgs` inputs, if
          > it reads other inputs, the testbed will fail to evaluate; e.g.,
          > `config` or `options`.
          >
          > This restriction was made for performance reasons. See `isEnabled`.
```

Personally I feel it makes more sense to use semantic wrapping here, since 80 falls so close to commas, etc? But up to you

---

> Personally I feel it makes more sense to use semantic wrapping here, since 80 falls so close to commas, etc? But up to you

In that case, I would simply improve the flow as follows:

```diff
           > [!CAUTION]
           >
-          > This option's value can only evaluate `lib` and `pkgs` inputs, if it
-          > reads other inputs, the testbed will fail to evaluate; e.g.,
-          > `config` or `options`.
-          >
-          > This restriction was made for performance reasons. See `isEnabled`.
+          > This option can only access `lib` and `pkgs` inputs. Attempting to
+          > read other inputs, like `config` or `options`, will cause the
+          > testbed evaluation to fail.
+          >
+          > This is a performance-driven restriction, as noted in `isEnabled`.
```